charts/kafka-ui/README.md

```diff
@@ -17,6 +17,6 @@ helm install kafka-ui kafka-ui/kafka-ui --set envs.config.KAFKA_CLUSTERS_0_NAME=
 ``` 
 To connect to Kafka-UI web application need to execute:
 ``` bash
-kubectl port-forward svc/kafka-ui 8080:80
+kubectl port-forward svc/kafka-ui 8080:8080
```

Comment:
Thanks for PR, it solves the #556, need to clarify that the k8s service use 80 port, and you still shall use `kubectl port-forward svc/kafka-ui 8080:80`. But your changes in NOTES.txt are correct because in case of port forwarding from POD we should use the 8080 port number.

Reply:
please restore to port 80